client/my-sites/stats/all-time-highlights-section/post-cards-group.tsx

```diff
@@ -100,10 +102,48 @@ export default function MostPopularPostCard( {
 			stripHTML( textTruncator( mostPopularPost?.title, POST_STATS_CARD_TITLE_LIMIT ) )
 		),
 		likeCount: mostPopularPost?.like_count,
-		viewCount: mostPopularPostViewCount, // or topViewedPost?.views
+		viewCount: topViewedPost?.views,
 		commentCount: mostPopularPost?.discussion?.comment_count,
 	};
 
+	const mobileCardsGroup = (
+		<DotPager>
+			<LatestPostCard siteId={ siteId } siteSlug={ siteSlug } />
+			{ ! isTopViewedPostRequesting && mostPopularPost && (
+				<PostStatsCard
+					heading={ translate( 'Most popular post in the past year' ) }
+					likeCount={ mostPopularPostData?.likeCount }
+					post={ mostPopularPostData }
+					viewCount={ mostPopularPostData?.viewCount }
+					commentCount={ mostPopularPostData?.commentCount }
+					titleLink={ `/stats/post/${ mostPopularPost.ID }/${ siteSlug }` }
+					uploadHref={
+						! isOdysseyStats ? `/post/${ siteSlug }/${ mostPopularPost.ID }` : undefined
+					}
+				/>
+			) }
```

Comment:
It might be good to refactor this out of the `mobileCardsGroup` and `cardsGroup` declarations to avoid regressions in the future (e.g., where we update one block but forget to update the other).

Something like this:

```js
	const cards = (
		<>
			<LatestPostCard siteId={ siteId } siteSlug={ siteSlug } />
			{ ! isTopViewedPostRequesting && mostPopularPost && (
				<PostStatsCard
					heading={ translate( 'Most popular post in the past year' ) }
					likeCount={ mostPopularPostData?.likeCount }
					post={ mostPopularPostData }
					viewCount={ mostPopularPostData?.viewCount }
					commentCount={ mostPopularPostData?.commentCount }
					titleLink={ `/stats/post/${ mostPopularPost.ID }/${ siteSlug }` }
					uploadHref={
						! isOdysseyStats ? `/post/${ siteSlug }/${ mostPopularPost.ID }` : undefined
					}
				/>
			) }
		</>
	);

	const mobileCardsGroup = (
		<DotPager>
			{ cards }
		</DotPager>
	);

	const cardsGroup = (
		<div className="highlight-cards-list">
			{ cards }
		</div>
	);
```

Reply:
Yes, I did miss it! 😅

Reply:
Yes! I did miss it. 😅 The DotPager seems only to accept the components array. [7b7442d](https://github.com/Automattic/wp-calypso/pull/74495/commits/7b7442dd1970dfbb353716ec163fe9d61908fd18)